controllers/sriovoperatorconfig_controller.go

```diff
@@ -522,3 +531,97 @@ func (r *SriovOperatorConfigReconciler) syncK8sResource(cr *sriovnetworkv1.Sriov
 	}
 	return nil
 }
+
+func (r *SriovOperatorConfigReconciler) syncOffloadMachineConfig(dc *sriovnetworkv1.SriovOperatorConfig) error {
```

Comment:
Shall we split the file renderring to two parts and sync the offload in sriov policy controller?
for example, one part is switchdev related configuration files (switchdev-configuration service, configure-switchdev script and NetworkManager service update)
the other part is ovs offload related configuration files (MCP, MC, ovs-vswitchd)

This way, if we were to add switchdev CNI support in the future, then it will not associated with the ovs-configuration and MCO changes.

Reply:
fixed.